java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S4790_java.html

```diff
@@ -151,3 +151,121 @@ <h2>Questionable Code Example</h2>
     }
 }
 </pre>
+<h2>Questionable Code Example</h2>
```

Comment:
This long `Questionable Code Example` section is duplicated, rule-api should be drunk.
I'm going to change `Questionable Code Example` by `Sensitive Code Example` in the RSPEC, it should fix rule-api.